### PR TITLE
Add condition for right click controls close VC-2619

### PR DESCRIPTION
**Pull request by nikitahl**

### CODE REVIEW

File: public/components/rightClickMenu/rightClickMenu.js [hunk 1]
```diff
@@ -45,13 +49,15 @@ export default class RightClickMenu extends React.Component {
     const targetElement = e.target
     let id = targetElement.getAttribute('data-vcv-element')
 
+    console.log('handleRightClick', id)
```
Comment (pavelthq) [inline public/components/rightClickMenu/rightClickMenu.js:52]: console.log

**Review by pavelthq — CHANGES_REQUESTED** [review]

File: public/components/rightClickMenu/rightClickMenu.js [hunk 2]
```diff
@@ -45,13 +49,15 @@ export default class RightClickMenu extends React.Component {
...
     if (!id) {
       const closest = targetElement.closest('[data-vcv-element]')
       if (closest) {
         id = closest.getAttribute('data-vcv-element')
       }
     }
 
+
```
Comment (pavelthq) [inline public/components/rightClickMenu/rightClickMenu.js:60]: extra enter?

**Review by pavelthq — CHANGES_REQUESTED** [review]

File: public/components/rightClickMenu/rightClickMenu.js (hunk 2 above)
Comment (nikitahl) [reply public/components/rightClickMenu/rightClickMenu.js]: Fixed

File: public/components/rightClickMenu/rightClickMenu.js (hunk 1 above)
Comment (nikitahl) [reply public/components/rightClickMenu/rightClickMenu.js]: Fixed

**Review by pavelthq — APPROVED** [review]

**Pull request merged**